CONTRIBUTING.rst

```diff
@@ -68,7 +68,9 @@ Development Workflow
          # Activate it
          conda activate networkx-dev
          # Install main development and runtime dependencies of networkx
-         conda install -c conda-forge --file requirements/default.txt --file requirements/test.txt --file requirements/developer.txt
+         conda install -c conda-forge --file requirements/default.txt
+	 conda install -c conda-forge --file requirements/test.txt
+	 pip install changelist==0.5
```

Comment:
Thanks for the PR @akshitasure12! This removes the dependencies listed in `requirements/developer.txt`. I wonder if it's just easier to add `changelist` to conda-forge than maintaining 2 different workflows. Hopefully this should build and be merged https://github.com/conda-forge/staged-recipes/pull/28089 :)